hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -268,6 +268,7 @@ public static boolean isReadOnly(
     case TenantGetUserInfo:
     case TenantListUser:
     case RangerBGSync:
+    case ListSnapshot:
```

Comment:
A comment just below this line is for RangerBGSync. I think it's better to move ListSnapshot one line up or move the comment above to RangerBGSync case.

Reply:
I fixed it.